config/locales/server.en.yml

```
@@ -1,9 +1,9 @@
 en:
   site_settings:
-    survey_monkey_enabled: "Enable to enable survey monkey plugin."
-    survey_monkey_survey_url: "Set to your specific survey monkey survey."
-    survey_monkey_manual_button: "Enable to add a button to topics to manually force send a survey"
-  survey_monkey:
+    survey_enabled: "Enable to enable survey plugin."
```

Enable to enable has an odd ring to it
```suggestion
    survey_enabled: "Enables survey plugin."
```

---

Not within the scope of this commit, will change in next pr.